lib/dynosaur/client/heroku_client.rb

```diff
@@ -0,0 +1,32 @@
+require 'platform-api'
+
+# A wrapper for the Heroku Platform API gem.
+module Dynosaur
+  module Client
+    module HerokuClient
+      def self.config
+        @config ||= Config.new
+      end
+
+      def self.configure
+        yield config if block_given?
+      end
+
+      def self.client
+        PlatformAPI.connect_oauth(api_key)
+      end
+
+      def self.app_name
+        config.app_name || fail('app_name must be set in the config')
```

Comment:
An orchestrator might be in charge of dynos on several Heroku apps, so putting this in the config is constraining vs initializing the client with it. Good expansion for later.